.openshift-ci/drivers/scripts/cpaas-support-package.sh

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+# shellcheck source=SCRIPTDIR/lib.sh
+source /scripts/lib.sh
+
+METADATA_DIR="/tmp/cpaas-support-packages/metadata"
+OUTPUT_DIR="/tmp/cpaas-support-packages/output"
+GCP_BASE_BUCKET=$1
+
+# Create the metadata directories
+for version in "/kernel-modules"/*/; do
+    md_version="$(basename "${version}")"
+    mkdir -p "${METADATA_DIR}/module-versions/$md_version"
+done
+
+target="${GCP_BASE_BUCKET}"
+
+if is_in_PR_context; then
+    BRANCH="$(get_branch)"
+    target="gs://stackrox-collector-modules-staging/pr-builds/${BRANCH}/${BUILD_ID}"
+fi
+
+/scripts/create-support-packages.sh \
+      /LICENSE \
+      "${target}" \
+      "${METADATA_DIR}" \
+      "${OUTPUT_DIR}"
+
+# Sync support packages
+gsutil -m rsync -r "${OUTPUT_DIR}" "${target}/cpaas"
+
+# Sleep for 120s
```

Comment:
I've seen this approach couple of times, the purpose here is to wait for syncing to finish, right? Maybe it's a good time to use some more reliable approach, what do you think?

Reply:
I believe this is just waiting for the sync to finish properly, it's been handled this way for some time now and it works reliably, we can look into making this better but in the interest of moving this PR forward I'd prefer to do so on a follow up.

Reply:
Sure, although it would be great to understand if this weird looking part is needed.